docs/Plugins.rst

```diff
@@ -1998,11 +1998,11 @@ Regrows all the grass. Not much to it ;)
 restrictice
 ===========
 Restrict traffic on all tiles on top of visible ice.
-See also `alltraffic`, `filltraffic`, and `restrictliquid`.
+See also `alltraffic`, `filltraffic`, and `restrictliquids`.
 
-.. _restrictliquid:
+.. _restrictliquids:
 
-restrictliquid
+restrictliquids
```

Comment:
Updating the line below would be good too (although I'm not sure if sphinx cares about that).